### PR TITLE
Added db schema & script

### LINKED ISSUE #39: Create PostGreSQL Script
# Description 

Create a `PostgreSQL` Script for all the tables and their relationship.

## Acceptable Criteria 

- Create a `PostgreSQL` script (including all the tables and relationships)
- Create a script to populate fake data

## Related Story 

Use the table naming and relationship from this #19  

## Technical Info

There are different ways of creating the script 
using [lucid diagram](https://www.lucidchart.com/pages/examples/er-diagram-tool#video)
or 
you can use the PostgreSQL tool

### PR DESCRIPTION
**Pull request by dinaolmelak**
# Description

This pull request contains a directory `resources` inside `sapweb` that contains `schema.sql` and `script.sql`. The schema file contains the script for creating the tables inside the database. The script file contains the fake data for populating the tables.

Fixes #39 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: sapweb/resources/schema.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+);
+
+CREATE TABLE "EVENTS" (
+  "events_id" int,
+  "mentor_id" int,
+  "student_id" int,
+  "startDate" Date,
+  "endDate" Date,
+  PRIMARY KEY ("events_id")
+);
```
Comment (kirubeltadesse) [inline sapweb/resources/schema.sql]: Add one more line at the end

File: sapweb/resources/script.sql [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+insert into "CALENDAR" ("calendar_id", "user_id", "assignedDate", "startDate", "endDate") values (33, 18, '2022-03-22', '2021-08-12', '2022-02-22');
+insert into "CALENDAR" ("calendar_id", "user_id", "assignedDate", "startDate", "endDate") values (34, 4, '2021-07-05', '2022-01-28', '2022-02-19');
+insert into "CALENDAR" ("calendar_id", "user_id", "assignedDate", "startDate", "endDate") values (35, 10, '2022-02-11', '2022-02-02', '2022-06-09');
+insert into "CALENDAR" ("calendar_id", "user_id", "assignedDate", "startDate", "endDate") values (36, 13, '2022-04-20', '2022-05-20', '2022-03-06');
+insert into "CALENDAR" ("calendar_id", "user_id", "assignedDate", "startDate", "endDate") values (37, 4, '2022-05-23', '2022-04-28', '2021-11-04');
+insert into "CALENDAR" ("calendar_id", "user_id", "assignedDate", "startDate", "endDate") values (38, 21, '2022-05-13', '2022-03-13', '2022-03-05');
+insert into "CALENDAR" ("calendar_id", "user_id", "assignedDate", "startDate", "endDate") values (39, 22, '2021-10-30', '2022-03-21', '2022-07-26');
+insert into "CALENDAR" ("calendar_id", "user_id", "assignedDate", "startDate", "endDate") values (40, 8, '2022-01-12', '2022-03-01', '2022-08-03');
```
Comment (kirubeltadesse) [inline sapweb/resources/script.sql]: add one more line at the end of the file

**Review by kirubeltadesse — APPROVED** [review]
Few comments. Nice Job 👍🏼

File: sapweb/resources/schema.sql (hunk 1 above)
Comment (dinaolmelak) [reply sapweb/resources/schema.sql]: I have gone ahead and added the new lines. 👍🏽

File: sapweb/resources/script.sql (hunk 2 above)
Comment (dinaolmelak) [reply sapweb/resources/script.sql]: Done 👍🏽

File: sapweb/resources/schema.sql [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  PRIMARY KEY ("calendar_id")
+);
+
+CREATE TABLE "USER" (
+  "user_id" int,
+  "fullName" VARCHAR,
+  "jNumber" int,
+  "email" VARCHAR,
+  "role" VARCHAR,
+  "userName" VARCHAR,
```
Comment (Getnet12) [inline sapweb/resources/schema.sql:28]: Is any specific reason why we have username? Can we not just use the email addresses for sign-in or signup purposes?

**Review by Getnet12 — APPROVED** [review]
This is great, Dinaol🫡

File: sapweb/resources/schema.sql (hunk 3 above)
Comment (kirubeltadesse) [reply sapweb/resources/schema.sql:28]: I can't think of a good use case for it right now. But we can use email addresses and passwords for singing or signup purpose. But every user has a username based on the UML diagram.

**Pull request merged**